app/src/main/java/com/example/drawingactivity/DrawViewModel.kt

```diff
@@ -31,37 +34,41 @@ class DrawViewModel : ViewModel() {
     var onUpdate: ((ViewState) -> Unit)? = null
 
     /**
-     * Invoked when a Layer is added to the List
-     * Takes in a String and returns nothing
-     * This informs the user on the result of adding a String to the list
+     * Invoked when a String is added to the List.
+     * Takes in a String and returns nothing.
+     * This informs the user on the result of adding a String to the list.
      */
-    var onAddLayerComplete: ((String?) -> Unit)? = null
+    var onAddToList: ((String?) -> Unit)? = null
 
     /**
-     * Invoked when a Layer name is changed the List
+     * Invoked when a String is changed the List.
+     * Confirms that the String in the list has correctly Changed.
      */
-    var onEditLayerComplete: ((String?, String?) -> Unit)? = null
+    var onEditList: ((String?, String?) -> Unit)? = null
```

Comment:
We need to be using layer objects, so let's not rename this

Reply:
Reverted the renamed layer functions and lambdas